server/models/product.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require("mongoose")
+
+const productSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  price: {
+    type: String,
```

Comment:
```suggestion
    type: Number,
```
Think it makes more sense for this to be a number